### DISCUSSION TITLE
mihomo 内核似乎有点慢

### DISCUSSION THREAD

**Discussion by fuwuqi298** [MetaCubeX/mihomo#1489] [General]
用了一段时间 mihomo，应用选的是 [clash-verge-rev](https://github.com/clash-verge-rev/clash-verge-rev)，用的机场没有变，但换到  [clash-verge-rev](https://github.com/clash-verge-rev/clash-verge-rev) 发现网站打开有点慢，相比 clash for windows，几乎秒开网站，感知挺明显的。

系统都是 windows10，dns 、网络都一样，都是内网，订阅链接也是一样的，远程桌面到 PC2
clash for windows
![image](https://github.com/user-attachments/assets/763616f5-b958-44d4-91ea-fc0392359004)
clash-verge-rev
![image](https://github.com/user-attachments/assets/6789af64-343d-4beb-baad-fdadd5039047)

两个应用设置都是默认的，就开启代理，延时测试链接两个应该是不一样，看了下延时，但差的有点多
![image](https://github.com/user-attachments/assets/d0f7f049-6336-45b4-9c60-c03bccf6f4f3)

有兴趣的可以关注看看~

**Comment by xiandu999-yu** [comment]
![image](https://github.com/user-attachments/assets/40c49e81-a9f8-41e4-9183-ee3cd9daa977)
这玩意儿开了吗？[开了后↓]
![image](https://github.com/user-attachments/assets/e4aa3af9-4b60-4e84-a292-af725ee94632)

**Reply by xiandu999-yu** [reply]
![image](https://github.com/user-attachments/assets/6ca65b27-56af-40ba-b09e-e174f5de91ab)
![image](https://github.com/user-attachments/assets/7daa97f2-8d04-4e15-ab4c-48612fa1509f)
图1开了，图2关了。

**Reply by fuwuqi298** [reply]
确实，我配置默认没有，应该是 false，加上后，延时一下就下来了，100 左右。

刚顺便看了下加载时间，clash-verge-rev 比 cfw 慢的原因是节点不一样，Google 搜索两个应用规则都是自动选择，但自动选择节点不一样。测了几遍，clash-verge-rev 都慢一些。

clash for windows 2s-7s，大部分 5s 内，clash-verge-rev 在 5-9s，从没进 5s 过。

![image](https://github.com/user-attachments/assets/f151bd9c-98da-4246-bb9a-b31c628e9c8c)
![image](https://github.com/user-attachments/assets/0542e2e1-6b7c-49e2-8eb1-9f7662b88901)


就图一乐，没啥。
